src/Nancy/Diagnostics/InteractiveDiagnostics.cs

```diff
@@ -175,7 +175,11 @@ private static string GetDescriptionFromProperty(IDiagnosticsProvider diagnostic
 
         private static string GetDescriptionFromAttribute(IDiagnosticsProvider diagnosticsProvider, MethodInfo methodInfo)
         {
+#if DOTNET5_4
+            var attribute = (DescriptionAttribute)methodInfo.GetCustomAttribute(typeof(DescriptionAttribute));
+#else
             var attribute = (DescriptionAttribute)Attribute.GetCustomAttribute(methodInfo, typeof(DescriptionAttribute));

```

Comment:
Can't this also be unified? There is a `GetCustomAttribute` method on `MethodInfo` on full-framework, if I am not mistaken?